### PR TITLE
Release 3.0.2 - Add module asg-ebs - Auto Scaling Group with EBS mount

### PR DESCRIPTION
**Pull request by dalenewby**
Release as version 3.0.2 unless adding a new module justifies a different version.

### CODE REVIEW

File: aws/asg-ebs/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+## Transitional Inputs
+
+ - `ebs_device` - Device name for EBS volume, e.g., "/dev/sdh"
+ - `ebs_mount_point` - Full path to directory on which to mount the file system contained in the EBS volume
+ - `ebs_vol_id` - EBS volume ID
+ - `ebs_mkfs_label` - Label for filesystem. Default: `Data`
+ - `ebs_mkfs_labelflag` - Flag preceding the label name in the mkfs command. Default: `-L`
+ - `ebs_mkfs_extraopts` - Extra options to pass to the mkfs command. Default: ""
+ - `ebs_fs_type` - Type of filesystem to create. Default: `ext4`
+
```
Comment (briskt) [inline aws/asg-ebs/README.md:37]: `ebs_mountopts` isn't listed here, should it be?

File: aws/asg-ebs/user-data.sh [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+echo "region = ${aws_region}" >> ~/.aws/config
+
+echo "[default]"                                  > ~/.aws/credentials
+echo "aws_access_key_id = ${aws_access_key}"     >> ~/.aws/credentials
+echo "aws_secret_access_key = ${aws_secret_key}" >> ~/.aws/credentials
+
+
+# Get my EC2 instance ID
+# From https://stackoverflow.com/questions/625644/how-to-get-the-instance-id-from-within-an-ec2-instance
+# INSTANCE_ID=`curl http://169.254.169.254/latest/meta-data/instance-id`
```
Comment (briskt) [inline aws/asg-ebs/user-data.sh]: Do you want this other method for getting the instance ID here for later reference?

**Review by briskt — DISMISSED** [review]
Didn't completely digest the script. Looks good, though.

File: aws/asg-ebs/README.md (hunk 1 above)
Comment (dalenewby) [reply aws/asg-ebs/README.md:37]: Yes, thanks.

File: aws/asg-ebs/user-data.sh (hunk 2 above)
Comment (dalenewby) [reply aws/asg-ebs/user-data.sh]: No, I don't need it and removed it.

**Review by briskt — APPROVED** [review]

**Pull request merged**